lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -431,13 +459,14 @@ export class FormFieldModel extends FormWidgetModel {
             }
             case FormFieldTypes.DATETIME: {
                 if (typeof this.value === 'string' && this.value === 'now') {
-                    this.value = moment(new Date()).utc().format(this.dateDisplayFormat);
```

Comment:
is UTC preserved here? probably not

Reply:
updated code to use parseISO() to preserve UTC info.